docs/README.md

```diff
@@ -2,137 +2,139 @@
+applications and managed services from your choice of cloud using `kubectl`.
 
 Crossplane can be installed into an existing Kubernetes cluster to add managed
 service provisioning or deployed as a dedicated control plane for multi-cluster
 management and workload scheduling.
 
 Crossplane enables the community to build and publish Stacks to add more clouds
 and cloud services to Crossplane with support for out-of-tree extensibility and
-independent release schedules. Crossplane includes Stacks for [GCP][stack-gcp], 
+independent release schedules. Crossplane includes Stacks for [GCP][stack-gcp],
 [AWS][stack-aws], and [Azure][stack-azure] today.
 
 <h4 align="center"><img src="media/crossplane-overview.png" alt="Crossplane"></h4>
 
 Crossplane has four main feature areas that can be used independently:
+
 1. Crossplane Services - provision managed services from kubectl.
 1. Crossplane Stacks - extend Crossplane with new functionality.
 1. Crossplane Workloads - define complete applications and schedule across
    clusters, regions, and clouds.
 1. Crossplane Clusters - manage multiple Kubernetes clusters from a single
    control plane.
 
-## Crossplane Services 
-Crossplane Services supports managed service provisioning
-using `kubectl`. It applies the Kubernetes pattern for Persistent Volume (PV)
-claims and classes to managed service provisioning with support for a strong
-separation of concern between app teams and cluster administrators. 
+## Crossplane Services
+
+Crossplane Services supports managed service provisioning using `kubectl`. It
+applies the Kubernetes pattern for Persistent Volume (PV) claims and classes to
+managed service provisioning with support for a strong separation of concern
+between app teams and cluster administrators.
 
 App teams can choose between cloud-specific and portable services including
 managed databases, message queues, buckets, data pipelines, and more to define
 complete applications, build once, and deploy into multiple clouds using
-continuous delivery pipelines or GitOps flows. 
+continuous delivery pipelines or GitOps flows.
 
 Cluster administrators can define self-service policies and best-practice
 configurations to accelerate app delivery and improve security, so app teams can
 focus on delivering their app instead of cloud-specific infrastructure details.
 
-The [Crossplane Services Guide][services-user-guide] explores related concepts and
-shows how to use Crossplane Services to deploy a Wordpress instance that
-securely consumes a managed `MySQLInstance` from GCP, AWS, or Azure from
-`kubectl`.
+Take a look at the [getting started guide][getting-started] for an introduction
+to Crossplane Services. The [Crossplane Services Guide][services-user-guide]
+also explores related concepts and shows how to use Crossplane Services to
+deploy a Wordpress instance that securely consumes a managed `MySQLInstance`
+from GCP, AWS, or Azure using `kubectl`.
```

Comment:
I still feel like our distinction between the stacks and services guides are somewhat hazy. To me, the difference is that in the services guide you bring your k8s cluster that you will run the application in and in stacks you provision it (i.e. make use of `KubernetesApplication`). Not sure if we can make that clearer here.